src/operator/numpy/np_elemwise_binary_op.cc

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file np_elemwise_binary_op.cc
+ * \brief CPU Implementation of numpy-compatible element-wise binary operations
+ */
+
+#include <mxnet/base.h>
+#include "../mshadow_op.h"
+#include "../operator_common.h"
+#include "../tensor/elemwise_binary_op.h"
+#include "../tensor/elemwise_binary_broadcast_op.h"
+
+namespace mxnet {
+namespace op {
+
+NNVM_REGISTER_OP(_np_bitwise_or)
+.set_num_inputs(2)
+.set_num_outputs(1)
+.set_attr<mxnet::FInferShape>("FInferShape", BinaryBroadcastShape)
+.set_attr<nnvm::FInferType>("FInferType", ElemwiseType<2, 1>)
```

Comment:
This may not be correct. At least, the official numpy does not support bitwise_or for floating numbers. Please add a special infer type function which gives the similar error message as the official numpy when input dtype is not supported.

Reply:
fixed in the most recent commit